common/params_keys.h

```diff
@@ -0,0 +1,120 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+inline static std::unordered_map<std::string, uint32_t> keys = {
+    {"AccessToken", CLEAR_ON_MANAGER_START | DONT_LOG},
```

Comment:
let's also move the DONT_LOG/etc definitions?

Reply:
This approach has a few downsides. We would need to rename params_keys.cc to params_keys..h and include it in params.h. This means that whenever keys is modified, every file that includes params.h would need to be recompiled,  Additionally, this would introduce a global keys variable into all files that include params.h, leading to potential namespace pollution.